opendebates/views.py

```diff
@@ -239,10 +239,15 @@ def vote(request, id):
 
         )
     )
+    previous_debate_time = get_previous_debate_time()
     if created:
         # update the DB with the real tally
         Submission.objects.filter(id=id).update(
             votes=F('votes')+1,
+            current_votes=F('current_votes')+(
+                1 if previous_debate_time is None or vote.created_at > previous_debate_time
```

Comment:
I was aiming for having the counts be immediately ready if and when the feature gets enabled.